.github/pull_request_template.md

```diff
@@ -26,3 +26,11 @@ _Optional: attachments like images, videos, etc._
 - [ ] Description is filled and free of optional paragraphs.
 - [ ] Adds/updates automated tests.
 
+---
+
+### UI accessibility checklist
+
+_If your PR includes UI changes, please utilize this checklist:_
+- [ ] Make sure you use the API for UI elements that support large fonts.
+- [ ] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
+- [ ] New UI elements have Accessibility strings for VoiceOver.
```

Comment:
Btw, what about putting the check already?
Most of the times the check should be there so it would save some time.
Or do we prefer it this way?

```suggestion
- [x] Make sure you use the API for UI elements that support large fonts.
- [x] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [x] New UI elements have Accessibility strings for VoiceOver.
```

Reply:
@caldrian I think for people who're aware it could be faster BUT it could lead also to situations where it's already ticked and people don't have to think about it, in other words consider it done already.

Reply:
I agree with @agisilaos . I wouldn't put the check by default so as not to ignore it.